lib/daffy_lib/caching_encryptor.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require 'rails/all'
```

Comment:
what if Rails is not defined? You are not guarding against this possibility in `lib/daffy_lib.rb` the way that you are for `daffy_lib/railtie`.

Reply:
Removed this and comments on similar lines.  They were copy/pasted from outstanding porky PR.